### DISCUSSION TITLE
Reviewing Pull Requests

### DISCUSSION THREAD

**Discussion by bountonw** [bountonw/translate#43] [General]
@matt @tonyquicktech @Deadpikle @hsteck1844 

This is the first time for me to be this involved in a collaborative project of this scale. Perhaps those with more experience can weigh in on the best way to manage reviewing pull requests. I have changed the settings to require at least one review before merging. Some of the pull requests are pretty mundane, `s/tpyo/typo/`, while others are actual code. As we do not have the fine controls that an organization would have, we are not able to limit who can do what on a particular subset of files. For this reason, I think it is good to have a review. Especially as we will be adding more people as collaborators as God opens the doors. What do you think and are there settings I don't know about that I should be using?

**Comment by tonyquicktech** [comment]
Agreed, as it at least adds an extra layer of protection.

**Comment by Deadpikle** [comment]
I am not really a part of this endeavor at this point, but I want to chime in since highlighted: Having a second pair of eyes on whatever you do is generally wise for the same reasons having someone else double-check your work is smart. However, if people's schedules for reviewing code start holding up the process of getting code integrated into the main branch, (e.g. there is only 1 person around for weeks at a time), then it can start preventing progress. However, given how there are several of you on here already and you seem fairly active, I don't believe this to be a real concern at this point.

So, tl;dr: looks good, good idea, and so on and so forth.
